README.md

```diff
@@ -1,6 +1,193 @@
 Job Restrictions Plugin for [Jenkins CI][1]
 =======================
-See [Job Restrictions Plugin Page][2] for more information.
+
+The plugin allows restricting job executions in order to change their
+behavior or to harden the security. With this plugin it is possible to
+configure nodes to accept only particular jobs. It is also possible to
+prevent job triggering by users and other jobs using various conditions.
+
+# About the plugin
```

Comment:
```suggestion
```

The paragraph above represents the About section. The header is not needed IMHO

Reply:
Change made.